CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for
+# review when someone opens a pull request.
+*       @NikoHadouken @volkramweber @steffenkleinle
```

Comment:
Do you know the github handle of Christian?

Reply:
His handle is @Stift